quickstart/kubernetes-guestbook.md

```diff
@@ -0,0 +1,373 @@
+---
+title: "Create a Guestbook with Redis and PHP"
```

Comment:
Here you call out PHP, below you call out Nginx.  Since PHP is mostly invisible to the authoring experience below, perhaps stick with Nginx here as well?

Reply:
I actually copied this from the Kubernetes quickstart on which I based this example (both the kubernetes.io and GKE docs refer to it under this name). I actually don't have a huge preference on this, so if -- knowing that I copied it -- you'd still prefer the rename, just lmk.